modules/standard/IO.chpl

```diff
@@ -8186,6 +8186,15 @@ proc fileWriter.writeln(const args ...?k, style:iostyle) throws {
   try this.writeHelper((...args), new ioNewline(), style=style);
 }
 
+@deprecated(notes="fileReader.flush is deprecated; it has no replacement because 'flush' has no effect on 'fileReader'")
+proc fileReader.flush() throws {
+  var err:errorCode = 0;
+  on this._home {
+    err = qio_channel_flush(locking, _channel_internal);
+  }
+  if err then try this._ch_ioerror(err, "in fileReader.flush");
+}
```

Comment:
Is there any reason to give this a body rather than just making it an empty procedure?

That said, if this body is identical to the one we had before, this is probably the more principled way to do it (it just seems odd to say "has no effect" and then to have it contain all this code... :) ).

Reply:
It's identical to what we had before